src/EnergyPlus/PluginManager.cc

```diff
@@ -411,23 +738,26 @@ namespace PluginManagement {
         std::string programDir = FileSystem::getParentDirectoryPath(programPath);
         std::string sanitizedProgramDir = PluginManager::sanitizedPath(programDir);
 
+        // so first things first is we need to link up with the Python DLL file and get refs to the functions in there
+        loadWrapperDLL();
+
         // I think we need to set the python path before initializing the library
         // make this relative to the binary
-        std::string pathToPythonPackages = sanitizedProgramDir + "/python_standard_lib";
+        std::string pathToPythonPackages = sanitizedProgramDir + DataStringGlobals::pathChar + "python_standard_lib";
         FileSystem::makeNativePath(pathToPythonPackages);
-        auto a = Py_DecodeLocale(pathToPythonPackages.c_str(), nullptr);
-        Py_SetPath(a);
-        Py_SetPythonHome(a);
+        wchar_t *a = (*EP_Py_DecodeLocale)(pathToPythonPackages.c_str(), nullptr);
+        (*EP_Py_SetPath)(a);
+        (*EP_Py_SetPythonHome)(a);
```

Comment:
These calls demonstrate the new way of making calls.  
- In the Python DLL itself, and the header, there are symbols and macros exposed like: `Py_SetPath`.
- In our PythonWrapper, we directly call those symbols and macros, but expose them as plain functions like: `EP_Py_SetPath`
- Then in this code, we have a reference to that `EP_Py_SetPath` function as a pointer
- To call it, we just deference the pointer and make the call, which is what the asterisk in the parentheses does.